src/scenes/Engagement/Delete/DeleteEngagement.tsx

```diff
@@ -19,7 +20,8 @@ import {
 } from './DeleteEngagement.graphql';
 
 type ProjectIdFragment =
-  | TranslationProjectIdFragment
+  | MomentumProjectIdFragment
+  | MultiplicationProjectIdFragment
   | InternshipProjectIdFragment;
```

Comment:
I'd move this type alias to a common spot. Probably in `common/fragments/` somewhere.